browser/src/control/Control.Menubar.js

```diff
@@ -693,6 +695,7 @@ L.Control.Menubar = L.Control.extend({
 				   {type: 'separator'},
 				   {name: _('Toggle UI Mode'), id: 'toggleuimode', type: 'action'},
 				   {name: _('Show Status Bar'), id: 'showstatusbar', type: 'action'},
+				   {name: _('Collapse Notebook Bar'), id: 'collapsenotebookbar', type: 'action'},
```

Comment:
For compact view we shouldn't add collapsenotebookbar menu item because we are now in the compact view.